src/routes/sse.ts

```diff
@@ -0,0 +1,123 @@
+import { type Request, type Response, Router } from "express"
+import { instance as createLogger } from "../utils/logger"
+import { validateBroadcastBody, validateSendRequest } from "../validation/sse"
+
+const router = Router()
+const logger = createLogger("routes.sse")
+
+const connections = new Map<string, Response>()
+const HEARTBEAT_INTERVAL_MS = 30000 //30seg
```

Comment:
This const looks like an env variable.
Put that in the config/envs.ts allowing it to be a env with a default value of 30000.